modules/model/src/main/scala/lucuma/itc/SpectroscopyTimeAndGraphResult.scala

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) 2016-2023 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package lucuma.itc
+
+import cats.data.NonEmptyList
+import cats.syntax.all.*
+import eu.timepit.refined.*
+import eu.timepit.refined.numeric.NonNegative
+import eu.timepit.refined.types.numeric.NonNegInt
+import eu.timepit.refined.types.numeric.PosInt
+import io.circe.CursorOp
+import io.circe.Decoder
+import io.circe.DecodingFailure
+import io.circe.Encoder
+import io.circe.HCursor
+import io.circe.Json
+import io.circe.generic.semiauto.*
+import io.circe.refined.*
+import lucuma.core.enums.*
+import lucuma.core.math.SignalToNoise
+import lucuma.core.syntax.string.*
+import lucuma.core.util.Enumerated
+import lucuma.core.util.TimeSpan
+import lucuma.itc.encoders.given
+import lucuma.itc.math.*
+
+case class SpectroscopyTimeAndGraphResult(
+  serverVersion:       String,
+  dataVersion:         String,
+  exposureTime:        TimeSpan,
+  exposures:           PosInt,
+  ccds:                NonEmptyList[ItcCcd],
+  charts:              NonEmptyList[ItcChart],
+  peakSNRatio:         SignalToNoise,
+  atWavelengthSNRatio: Option[SignalToNoise]
+) derives Encoder.AsObject
```

Comment:
Could/should this share anything with the normal spectroscopy result?

Reply:
Right, we could extend one from the other and maybe encode it "correctly"

Reply:
Or just have the graph result include the spectroscopy result like the input.  Maybe it doesn't make sense.